docker-compose.yml

```diff
@@ -5,14 +5,14 @@ version: '3'
 services:
 
   mongodb:
-    image: mvertes/alpine-mongo
+    image: mongo:4.4.9
     container_name: mongodb
-    ports:
-      - '27017:27017'
-    expose:
-      - '27017'
     networks:
       - scout-net
+    ports:
+      - '27013:27013'
+    expose:
+      - '27013'
```

Comment:
I changed this port so it doesn't mess up with an eventual database hosted on the user machine

Reply:
Nice idea, but you dont really change the port that the mongod answers to? That would be possible with options to the startup, or a custom mongod.conf for the container. But can I suggest just to leave it on 27017 in the container, remap it to the host 27013 for any debugging, and expose the container 27017 to the compose network.

Reply:
Like so with these changes: 
![Screenshot 2021-10-08 at 11 07 02](https://user-images.githubusercontent.com/758570/136530014-09b53544-d514-49af-a75e-bdcc64b77346.png)
![Screenshot 2021-10-08 at 11 07 12](https://user-images.githubusercontent.com/758570/136530018-65048ba4-9921-4caf-bf18-d18e8e9125e8.png)

Reply:
Good suggestion!! 👍🏻

Reply:
It was actually what I wanted to do but I did it wrong in the PR :)